Modules/symtablemodule.c

```diff
@@ -71,13 +71,57 @@ static PyMethodDef symtable_methods[] = {
     {NULL,              NULL}           /* sentinel */
 };
 
+static int
+symtable_init_stentry_type(PyObject *m)
+{
+    return PyType_Ready(&PySTEntry_Type);
```

Comment:
I don't see the value of having two exec slots. Why not simply putting all exec code into a single function?

Reply:
They do different things. Also ``PySTEntry_Type`` is define somewhere else and we might want to move the type initialization when the ``PySTEntry_Type`` is refactored.